src/components/gameComponent/gameComponent.js

```diff
@@ -1,37 +1,42 @@
-// import React, { Component } from 'react';
-// import Board from '../boardComponent';
+import React, { Component } from 'react';
+import Board from "../boardComponent/boardComponent";
+import logo from '../../images/game-logo3.png';
 
-// class Game extends Component {
 
-    // constructor(props) {
-    //     super(props) // call super parent class constructor props
-    //     this.state = {show: false} // if true the data will be shown
-    //     // this.toggleDiv = this.toggleDiv.bind(this); // bind this to the toggle div
-    // }
+class Main extends Component {
 
-    // toggleDiv = (e) => {
-    //     e.preventDefault();
-    //     console.log("Start Button Clicked");
-    //     const { show } = this.state;
-    //     this.setState({show: !show}) 
-    //     // this.setState = { show: !show} // change values in state attribute
-    //     console.log(this.state);
-    // }
-//     render() {
-//         return (
-//             <div className="Game">
-//                 {/* <div className="start_game">
-//                     <button onClick={this.toggleDiv} type="button" className="btn btn-outline-primary">Start Game</button> 
-//                 </div> */}
-//                 <div className="Game_Board">
-//                 {/* //based on state variable the board will show */}
-//                     {this.state.show && <Board /> } 
-//                 </div>
-//             </div>
-//         );
-//     }
+        constructor(props) {
+        super(props) // call super parent class constructor props
+        this.state = {
+            show: false // if true the data will be shown
+        
+        } 
+    }
 
-// }
+    toggleDiv = (e) => {
+        e.preventDefault();
+        console.log("Start Button Clicked!");
+        const { show } = this.state;
+        this.setState({show: !show}) 
+        // this.setState = { show: !show} // change values in state attribute
+        console.log(this.state);
```

Comment:
as a general rule, make sure to skim your branch for any console logs and remove them before pushing up. We should never be pushing debugging console logs to `master`